crates/bestool/tests/cli_tests.rs

```diff
@@ -110,3 +117,73 @@ fn stop_db(temp_dir: TempDir) -> Result<()> {
 
 	Ok(())
 }
+
+pub fn find_postgres_bin(name: &str) -> Result<OsString> {
```

Comment:
is this copied from the lib? can we expose that and re-use instead?

Reply:
Oh, I missed I can link the `test` crate to `bestool` because trycmd doesn't use that. It's fixed now.